### PR TITLE
Split election drop stats to be either overflow or expired

### PR DESCRIPTION
**Pull request by MajorChump**
At the moment it's not clear if elections are simply expiring after 5 minutes or being dropped due to the container overflowing. This should give us a better insight and allow us to monitor the situation.

### CODE REVIEW

File: nano/node/active_transactions.cpp [hunk 1]
```diff
@@ -330,6 +330,10 @@ void nano::active_transactions::request_confirm (nano::unique_lock<nano::mutex>
 			}
 
 			// Locks active mutex, cleans up the election and erases it from the main container
+			if (!election_l->confirmed())
```
Comment (theohax) [inline nano/node/active_transactions.cpp]: PR looks good! Don't forget to run `ci/clang-format-all.sh` so that formatting rules get applied.

**Review by theohax — DISMISSED** [review]
LGTM!

File: nano/node/active_transactions.cpp [hunk 2]
```diff
@@ -330,6 +330,10 @@ void nano::active_transactions::request_confirm (nano::unique_lock<nano::mutex>
 			}
 
 			// Locks active mutex, cleans up the election and erases it from the main container
+			if (!election_l->confirmed ())
```
Comment (SergiySW) [inline nano/node/active_transactions.cpp]: Can it be `confirmed_l` ?

File: nano/node/active_transactions.cpp [hunk 3]
```diff
@@ -316,6 +316,7 @@ void nano::active_transactions::request_confirm (nano::unique_lock<nano::mutex>
 	for (auto const & election_l : elections_l)
 	{
 		bool const confirmed_l (election_l->confirmed ());
+		unconfirmed_count_l += !confirmed_l;
```
Comment (MajorChump) [inline nano/node/active_transactions.cpp:319]: This is needed for logging on L349

**Review by clemahieu — APPROVED** [review]

**Pull request merged**